### PR TITLE
DP-21793: Add character countdown and warning when page titles exceed 70 characters

### PR DESCRIPTION
**Pull request by arthurbaghdas**
**Description:**
Patched Maxlength module, configured fields on the site.

**Jira:** (Skip unless you are MA staff)
https://jira.mass.gov/browse/DP-21793

**To Test:**
- [ ] Try adding content of each type and see how maxlength module behaves.

### CODE REVIEW

File: conf/drupal/config/core.entity_form_display.node.advisory.default.yml [hunk 1]
```diff
@@ -83,7 +85,6 @@ third_party_settings:
       weight: 13
       format_type: tab
       format_settings:
-        label: Details
```
Comment (weekbeforenext) [inline conf/drupal/config/core.entity_form_display.node.advisory.default.yml]: This seems unrelated. It doesn't seem to make a difference when looking at the form, as far as I can tell though.

File: conf/drupal/config/core.entity_form_display.node.advisory.default.yml [hunk 2]
```diff
@@ -113,7 +114,6 @@ third_party_settings:
       weight: 11
       format_type: fieldset
       format_settings:
-        label: '  '
```
Comment (weekbeforenext) [inline conf/drupal/config/core.entity_form_display.node.advisory.default.yml]: This might also be unrelated.

**Review by weekbeforenext — APPROVED** [review]
I noted a couple of odd config changes, but this looks good. If the config changes I commented on are expected, this is good to merge after resolving conflicts.

File: conf/drupal/config/core.entity_form_display.node.advisory.default.yml (hunk 1 above)
Comment (arthurbaghdas) [reply conf/drupal/config/core.entity_form_display.node.advisory.default.yml]: Fixed, thanks

File: conf/drupal/config/core.entity_form_display.node.advisory.default.yml (hunk 2 above)
Comment (arthurbaghdas) [reply conf/drupal/config/core.entity_form_display.node.advisory.default.yml]: Fixed, thanks

**Pull request merged**